src/components/exploreFurther/exploreFurther.tsx

```diff
@@ -74,6 +74,7 @@ const ExploreFurther = () => {
                 rel="noreferrer noopener"
                 sx={{
                   textDecoration: "none !important",
+                  _hover: { textDecoration: "underline 1px dotted !important" },
```

Comment:
This works. Have you tried the `url` prop in the `CardHeading` component? I think that should work but haven't checked. It might allow you to remove lines 76 and 77.

Reply:
Ah gotcha, I misread that at first.